jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/HttpExchange.java

```diff
@@ -174,22 +175,23 @@ private boolean completeRequest(Throwable failure)
 
     public boolean isResponseComplete()
     {
-        try (AutoLock l = lock.lock())
+        try (AutoLock ignored = lock.lock())
         {
             return responseState == State.COMPLETED;
         }
     }
 
     public boolean responseComplete(Throwable failure)
     {
-        try (AutoLock l = lock.lock())
+        try (AutoLock ignored = lock.lock())
         {
             return completeResponse(failure);
         }
     }
 
     private boolean completeResponse(Throwable failure)
     {
+        assert lock.isHeldByCurrentThread();
```

Comment:
Can you rename the method to `completeResponseLocked` ?

Reply:
Done.